atlp-rwanda/phantom-fe-codebandits
# #CP-29 feat:  crud operations for routes integration

{::comment} # _Title_: #162836478 Create the signup feature {:/comment}
{::comment} _description_{:/comment}

### What does this PR do?

{::comment} Explain the why/what not the how {:/comment}

{::comment} [what the PR does] {:/comment}

### Description of Task to be completed

- [ ] Task 3

### How should this be manually tested?

{::comment} Remember to add the code block {:/comment}

```
git clone https://github.com/atlp-rwanda/phantom-be-codebandits
cd phantom-be-codebandits
git fetch origin
git checkout <your branch>
npm install
//configure the .env according to the .env.sample file
// start the database
npm run dev
// open browser or postman (Check linked public workplace for testing)
```

### Any background context you want to provide?

{::comment} Explain the packages you may have used, some unusual coding patterns you have added and so on {:/comment}

### What are the relevant pivotal tracker stories?

{::comment} ["<Trello card id>"](<"Trello card url">) {:/comment}

### Screenshots (if appropriate)
